index.js

```diff
@@ -247,29 +252,39 @@ function toggleParticipantsBar() {
 }
 
 /* Other helper functions */
+
+// Starts an interval to check local network info
+// every 2 seconds.
+function startNetworkInfoPing() {
+  networkUpdateID = setInterval(() => {
+    updateNetworkInfoDisplay();
+  }, 2000);
+}
+
 // Populates 'network info' with information info from daily-js
 async function updateNetworkInfoDisplay() {
   const videoSend = document.getElementById('video-send'),
     videoReceive = document.getElementById('video-receive'),
-    packetSend = document.getElementById('packet-send'),
-    packetReceive = document.getElementById('packet-receive');
-
-  let statsInfo = await callFrame.getNetworkStats();
+    videoPacketSend = document.getElementById('video-packet-send'),
+    videoPacketReceive = document.getElementById('video-packet-receive');
 
+  const statsInfo = await callFrame.getNetworkStats();
+  const stats = statsInfo.stats;
+  const latest = stats.latest;
   videoSend.innerHTML = `${Math.floor(
-    statsInfo.stats.latest.videoSendBitsPerSecond / 1000
+    latest.videoSendBitsPerSecond / 1000,
```

Comment:
pretty sure we don't want /1000 here (or below) because the value is already in bits/second. This indicates that maybe it used to think it was ms?

Reply:
oh geez. nm. i see below it's to convert and report in kb/s. ignore me.